client/src/app/services/manage-members.service.ts

```diff
@@ -15,4 +15,9 @@ export class ManageMembersService {
   getAllClients(): Observable<any> {
     return this.http.get<User[]>(`${this.BASEURL}/getAllClients/`);
   }
+
+  sendNameToObtainClients(name): Observable<any>
```

Comment:
I feel the method name is a bit weird, you can just call it getSpecificClients(name) or something around that.

Reply:
Fixed in commit 38fa446370aa1c6aa2e3d4df0f78677c0e8076b2